tests/integration/test_charm.py

```diff
@@ -110,6 +153,40 @@ async def test_grafana_dashboards_devices(ops_test: OpsTest, mocker):
     await assert_grafana_dashboards(app, dashboards)
 
 
+async def test_loki_alert_rules_devices(ops_test: OpsTest, mocker):
+    """Test Loki alert rules for devices are defined in relation data bag."""
+    async with ops_test.fast_forward():
+        await ops_test.model.wait_for_idle(apps=[APP_NAME], status="active", timeout=120)
+    app = ops_test.model.applications[APP_NAME]
+    alert_rules = get_alert_rules(LOKI_ALERT_RULES_DIRECTORY)
```

Comment:
It is confusing to have both `get_alert_rules` & `_get_alert_rules` . Care alias'ing one (`import as`) and adding a couple comments?